src/internal/templates/internal/header.html

```diff
@@ -38,6 +38,9 @@
     <a class="item" href="{% host_url 'index_page' host 'main' %}">
         <div class="text">{% translate "Public pages" %}</div>
     </a>
+    <a class="item" href="{% host_url 'admin_panel' host 'main' %}">
+        <div class="text">{% translate "Administration" %}</div>
+    </a>
```

Comment:
From a UX perspective, I don't think this button should be in this menu, as it can easily be confused to mean "Administration of the internal pages (i.e. the site I'm currently on)" - given that we've been able to successfully create a mental separation between the public and internal site with our users, of course. And as it is now, it's just one extra click to get to the public pages :)

But if you all agree that the button should be there, I'd suggest wrapping it in the same check as is done on the public pages, so that unprivileged users are not met with a 403 page 🙂
```suggestion
    {% if user|can_view_admin_panel %}
        <a class="item" href="{% host_url 'admin_panel' host 'main' %}">
            <div class="text">{% translate "Administration" %}</div>
        </a>
    {% endif %}
```